lib/timeline/locales.js

```diff
@@ -1,7 +1,8 @@
 // English
 exports['en'] = {
   current: 'current',
-  time: 'time'
+  time: 'time',
+  deleteItemButton: 'Delete this item',
```

Comment:
I think we now have to update the other translations as well!

Reply:
Used localization texts from vis-network in 
https://github.com/visjs/vis-timeline/pull/88/commits/7b38cd5789e5152ed5024b8a54c9d7c3ade69688.

And added Japanese localizaiton in https://github.com/visjs/vis-timeline/pull/88/commits/cae132e68ec0da02b834a449d545e8e866d8d351